va/va_trace.c

```diff
@@ -538,6 +556,19 @@ static void FILE_NAME_SUFFIX(
     if (left < (size + 8 + 10))
         return;
 
+#if defined(_WIN32)
+    SYSTEMTIME st;
+    GetSystemTime(&st);
```

Comment:
The `compat_win32.h` should probably provide a `gettimeofday()` compatible implementation. Adding multiple WIN32 snippets through the code doesn't look nor scale well.